gc/base/SparseAddressOrderedFixedSizeDataPool.cpp

```diff
@@ -382,6 +408,19 @@ MM_SparseAddressOrderedFixedSizeDataPool::updateSparseDataEntryAfterObjectHasMov
 	return ret;
 }
 
+bool
+MM_SparseAddressOrderedFixedSizeDataPool::updateAndVerifySparseDataEntryAfterObjectHasMoved(void *dataPtr, void *proxyObjPtr, uintptr_t size, void *newProxyObjPtr)
```

Comment:
for symmetry, let's call use oldProxyObjPtr

Reply:
update at in all other spots (method declarations and in SparseVirtualMemory class)